templates/items/details/details-feat.hbs

```diff
@@ -0,0 +1,38 @@
+<fieldset>
+    <legend>{{ localize "DND5E.ItemFeatureDetails" }}</legend>
+
+    {{!-- Feature Type --}}
+    {{ formField fields.type.fields.value value=source.type.value choices=config.featureTypes labelAttr="label" blank=""
+                 localize=true }}
+
+    {{!-- Feature Sub-Type --}}
+    {{#if itemSubtypes}}
+    {{ formField fields.type.fields.subtype value=source.type.subtype choices=itemSubtypes blank="" localize=true
+                 label=(localize "DND5E.ItemFeatureSubtype" category=(lookup (lookup config.featureTypes source.type.value) "label")) }}
+    {{/if}}
+
+    {{!-- Feature Prerequisites --}}
+    {{ formField fields.prerequisites.fields.level value=source.prerequisites.level localize=true step=1
+                 label="DND5E.Prerequisites.FIELDS.prerequisites.level.label"
+                 hint="DND5E.Prerequisites.FIELDS.prerequisites.level.hint" }}
+
+    {{!-- Feature Properties --}}
+    {{ formField fields.properties options=properties.options label="DND5E.ItemFeatureProperties" localize=true
+                 input=inputs.createMultiCheckboxInput stacked=true classes="checkbox-grid checkbox-grid-3" }}
+</fieldset>
+
+{{#if system.isEnchantmentSource}}
+<fieldset>
+    <legend>{{ localize "DND5E.Enchantment.Label" }}</legend>
+
+    {{!-- Max Enchantments --}}
+    {{ formField fields.enchantment.fields.items.fields.max value=source.enchantment.items.max localize=true
+                 label="DND5E.Enchantment.FIELDS.enchantment.items.max.label"
+                 hint="DND5E.Enchantment.FIELDS.enchantment.items.max.hint" }}
+
+    {{!-- Enchantment Replacement --}}
+    {{ formField fields.enchantment.fields.items.fields.period value=source.enchantment.items.period localize=true
+                 choices=config.enchantmentPeriods labelAttr="label" blank="DND5E.UsesPeriods.Never"
+                 hint="DND5E.Enchantment.FIELDS.enchantment.items.period.hint" }}
```

Comment:
Getting an error from these because `system.enchantment` was part of `ActionTemplate` that is no longer included in `FeatData`. Since the only thing left from that are these two fields which are only used on features, we'll need to move `items.max` and `items.period` over to `FeatData` (perhaps migrating to a new path like `system.enchantment.max` and `system.enchantment.period`).

Reply:
Used `enchant` since there's a class with the `enchantment` shim.